src/types/Config.tsx

```diff
@@ -592,117 +608,117 @@ export const ALL_PAGES: IPage[] = [
   },
   {
     name: PAGES.COMPANY_DATA,
-    role: ROLES.MY_ORGANIZATION_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.MY_ORGANIZATION_VIEW),
     element: <CompanyData />,
   },
 ]
 
-export const ALL_OVERLAYS: IOverlay[] = [
+export const ALL_OVERLAYS: RestrictedItem[] = [
   {
     name: OVERLAYS.ADD_BPN,
-    role: ROLES.MODIFY_USER_ACCOUNT,
+    allowTo: () => userHasPortalRole(ROLES.MODIFY_USER_ACCOUNT),
   },
   {
     name: OVERLAYS.ADD_USER,
-    role: ROLES.USERMANAGEMENT_ADD,
+    allowTo: () => userHasPortalRole(ROLES.USERMANAGEMENT_ADD),
   },
   {
     name: OVERLAYS.ADD_MULTIPLE_USER,
-    role: ROLES.USERMANAGEMENT_ADD,
+    allowTo: () => userHasPortalRole(ROLES.USERMANAGEMENT_ADD),
   },
   {
     name: OVERLAYS.ADD_TECHUSER,
-    role: ROLES.TECHUSER_ADD,
+    allowTo: () => userHasPortalRole(ROLES.TECHUSER_ADD),
   },
   {
     name: OVERLAYS.DELETE_TECHUSER,
-    role: ROLES.TECHUSER_DELETE,
+    allowTo: () => userHasPortalRole(ROLES.TECHUSER_DELETE),
   },
   {
     name: OVERLAYS.ADD_APP_USER_ROLES,
-    role: ROLES.MODIFY_USER_ACCOUNT,
+    allowTo: () => userHasPortalRole(ROLES.MODIFY_USER_ACCOUNT),
   },
   {
     name: OVERLAYS.EDIT_APP_USER_ROLES,
-    role: ROLES.MODIFY_USER_ACCOUNT,
+    allowTo: () => userHasPortalRole(ROLES.MODIFY_USER_ACCOUNT),
   },
-  { name: OVERLAYS.APP, role: ROLES.APPSTORE_VIEW },
+  { name: OVERLAYS.APP, allowTo: () => userHasPortalRole(ROLES.APPSTORE_VIEW) },
   { name: OVERLAYS.NEWS },
   {
     name: OVERLAYS.PARTNER,
-    role: ROLES.PARTNER_NETWORK_VIEW,
+    allowTo: () => userHasPortalRole(ROLES.PARTNER_NETWORK_VIEW),
```

Comment:
@oyo  here we need the BPDM role inside the user account.
<img width="209" alt="image" src="https://github.com/eclipse-tractusx/portal-frontend/assets/94133633/f88d44b4-5dac-42c4-be4a-4592a7ef6b94">